content/includes/footer.html

```diff
@@ -33,6 +33,9 @@
       TTY:&nbsp;<a href="tel:+18008778339" onClick="footerNavClick(this);">1-800-877-8339</a><br>
 Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (<abbr title="eastern time">ET</abbr>)</li>
       <li><p><button class="va-overlay-trigger usa-button-unstyled va-footer-vcl-trigger" data-show="#modal-crisisline">Get help from the Veterans Crisis Line</button></p></li>
+    </br>
+      <li><h4 class="va-footer-linkgroup-title">New to VA?</h4></li>
```

Comment:
Shouldn't this be `<br/>`?

Reply:
Blargh. `<br>` just as it is works fine. reverting to that.